src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js

```diff
@@ -0,0 +1,111 @@
+(function(global, doc, eZ, CKEditor) {
+    class BaseRichText {
+        constructor(config) {
+            this.ezNamespace = 'http://ez.no/namespaces/ezpublish5/xhtml5/edit';
```

Comment:
ibexa?

Reply:
This is what our backend expects and this is not that easy to change. We would have to also change on the backend and probably prepare some migration for previous versions. Definitely out of the scope of this PR.